docs/reference/api/pytorch.txt

```diff
@@ -170,6 +170,18 @@ your code:
    to the ``clip_grads`` argument of ``step_optimizer``
    while removing the ``PyTorchCallback`` counterpart in the :meth:`build_callbacks` method.
 
+Debugging FAQ
+-------------
+
+* **Q: If my job got paused (e.g. if a higher weight job gets scheduled over the top of a running job),
+  then the resulting validation accuracies are significantly lower than for jobs that weren't paused,
+  even though the training accuracies are the same.**
```

Comment:
I know that we talked about adding an FAQ earlier, but now that I read this, I think this is way too specific.

Maybe we could rephrase the question to be something more generic, like:
```
Q: Why am I seeing strange metrics for trials which are paused and later continued (jumps in loss, worse convergence, etc)?
```